src/chapters/orm-relationships/many-to-one.rst

```diff
@@ -0,0 +1,205 @@
+-------------------------------------
+
+The steps needed to implement ``AbstractEntity`` are outlined in :ref:`orm1-studio`.
+
+Setting Up the Relationship - Video
+-----------------------------------
+
+We are now ready to create a relationship between ``Event`` and ``EventCategory``.
+
+.. raw:: html
+
+   <div style="text-align:center;"><iframe width="800" height="450" src="https://www.youtube.com/embed/aFr_E2T7zZ8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
+
+The final code from this video is in the `many-to-one branch <https://github.com/LaunchCodeEducation/coding-events/tree/many-to-one>`__ of ``coding-events``.
+
+Setting Up the Relationship - Text
+----------------------------------
+
+Our many-to-one relationship is established in the ``Event`` class; each event will have one category, and there may be many events in a given category. 
+
+Currently, similar functionality is enabled via the ``EventType`` field of ``Event``. However, ``EventType`` is an enum, which means that new values can not be added without changing the code and re-compiling. Using the persistent ``EventCategory`` class to organize events will be a much more flexible and user-friendly approach. 
+
+Replacing ``EventType`` With ``EventCategory``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the ``Event`` class, replace the ``type`` field with a new private field of type ``EventCategory``:
+
+.. sourcecode:: java
+   :lineno-start: 25
+
+   private EventCategory eventCategory;
+
+This will introduce compiler errors wherever ``type`` is referenced. We will fix these shortly.
+
+First, however, update ``Event`` so that:
+
+#. Its main constructor (i.e. NOT the no-arg constructor) takes a parameter of type ``EventCategory`` and uses it to initialize ``eventCategory``.
+#. It has a getter/setter pair for ``eventCategory``.
+#. The ``eventCategory`` field has the ``@NotNull`` validation annotation, along with an appropriate message.
+
+This establishes a relationship between the two classes, but that relationship is not yet *persistent*. To configure Hibernate to keep track of this relationship in the database, we add the ``@ManyToOne`` annotation to ``eventCategory``. 
+
+.. sourcecode:: java
+   :lineno-start: 27
+
+   @ManyToOne
+   @NotNull(message = "Category is required")
+   private EventCategory eventCategory;
+
+This annotation informs Hibernate that there can be many events for each category, but only one category per event. 
+
+Before we can start up the app and test our changes, we need to remove the code that references the deleted ``type`` field. A quick way to find these references is to find ``EventType`` in the *Project* pane of IntelliJ, right-click, and select *Find Usages*. This will open a pane listing each location ``EventType`` occurs in our project, whether that is an object of that type or a direct reference to the class itself. 
```

Comment:
Missing word fix: "...each location where ``EventType`` occurs in our project..."

Alternatively, "This opens a pane showing every occurrence of ``EventType`` in our project..."